client/settings/index.js

```diff
@@ -43,18 +41,6 @@ window.addEventListener( 'load', () => {
 	enqueueFraudScripts( wcpaySettings.fraudServices );
 } );
 
-const settingsContainer = document.getElementById(
-	'wcpay-account-settings-container'
-);
-if ( settingsContainer ) {
-	ReactDOM.render(
-		<WCPaySettingsContext.Provider value={ wcpaySettings }>
-			<SettingsManager />
-		</WCPaySettingsContext.Provider>,
-		settingsContainer
-	);
-}
-
```

Comment:
Sorry for the confusion in my comment - this is needed.
The parts that are not needed are
https://github.com/Automattic/woocommerce-payments/blob/487284e6f4bce31fb3382fa8aba816df841a3477/client/settings/index.js#L16-L40
https://github.com/Automattic/woocommerce-payments/blob/487284e6f4bce31fb3382fa8aba816df841a3477/client/settings/index.js#L70-L181

Reply:
Whoops! My mistake. I've added that bit back.

Reply:
I'm still seeing the settings code that is unrelated to the newer settings.
You'll need to remove L16-40 and L70-181 from `woocommerce-payments/client/settings/index.js`

Reply:
> L16-40 and L70-181 from woocommerce-payments/client/settings/index.js
Yikes. I've really misunderstood what you were referencing, then! I've removed those lines as well as line 6 as `__` was no longer used.